frontend/map.component.ts

```diff
@@ -1,23 +1,59 @@
 import { Component, OnInit } from "@angular/core";
 
-import { Tweet } from "./tweet";
 import { MapService } from "./map.service";
+import { MapObject } from "./mapObject";
+import { Tweet } from "./tweet";
 
 @Component({
     selector: "map-app",
     templateUrl: "map.component.html",
     providers: [MapService]
 })
 export class MapComponent implements OnInit {
+
+    map: google.maps.Map;
+    currentlyOpenInfoWindow: google.maps.InfoWindow = null;
```

Comment:
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined